docs/sources/developer/Developing-Windows.md

```diff
@@ -1,55 +1,68 @@
 # Developing on Windows
 
-To download the latest version of Plaso you'll need to install the git tools: http://git-scm.com/downloads
+## Git
+To download the latest version of Plaso you'll need to install the 
+[git tools](http://git-scm.com/downloads).
 
 Checkout the plaso source from the git repo:
 ```
 git clone https://github.com/log2timeline/plaso.git
 ```
 
-**If you intend to submit code make sure to configure git to use convert to the Unix-style end-of-line characters (linefeed) on submission and not have the Windows-style end-of-line characters (carriage return + linefeed).** 
+**If you intend to submit code make sure to configure git to use convert to the 
+Unix-style end-of-line characters (linefeed) on submission and not have the 
+Windows-style end-of-line characters (carriage return + linefeed).** 
 
-We recommend to configure your editor of choice to use linefeed only and turn off git's autocrlf:
+We recommend to configure your editor of choice to use linefeed only and turn 
+off git's autocrlf:
 ```
 git config --global core.autocrlf false
 ```
 
-To be able to run the plaso [development release](Releases-and-roadmap.md) on Windows you'll have to have installed the [dependencies](Dependencies.md).
+## Dependencies
 
-Check if you have all the dependencies installed and have the right minimum version:
+Install the dependencies, using the 
+[pre-packaged dependencies](Dependencies.html#Windows).
+
+Check if you have all the dependencies installed and have the right minimum 
+version:
 ```
 C:\Python27\python.exe utils\check_dependencies.py
 ```
 
-**Note that some dependencies are actively under development and can be frequently updated, therefore we recommend checking the status of the dependencies regularly.**
+**Note that some dependencies are actively under development and can be 
+frequently updated, therefore we recommend checking the status of the 
+dependencies regularly.**
 
-## Running the development release
-To run the development release directly from source make sure Python can find the plaso source files by setting PYTHONPATH correspondingly.
+## Running the development code
+To run the development code directly from source make sure Python can find the 
+Plaso source files by setting PYTHONPATH correspondingly.
 ```
 set PYTHONPATH=C:\plaso-build\plaso
 ```
 
-To run e.g. pinfo:
+To run a tool, for example `pinfo`:
 ```
-C:\Python27\python.exe C:\plaso-build\plaso\plaso\frontend\pinfo.py plaso.db
+C:\Python27\python.exe C:\plaso-build\plaso\plaso\frontend\pinfo.py test.plaso
 ```
 
 ## Development tools
-If you intend to do development on plaso you'll also need to install some development tools:
+If you intend to do development on plaso you'll also need to install some 
+development tools:
 
 * PyLint
 * Python Mock
 
 ### PyLint
```

Comment:
Maybe replace sections PyLint and Python Mock, by instructions on how to install?

Reply:
I'd like to overhaul all these instructions, but I'll do that in a later change.

Reply:
ack